src/components/Search/SearchAutocomplete.tsx

```diff
@@ -73,8 +79,10 @@ const SearchAutocomplete: React.FC<{
       renderOption={option => {
         return <MenuItem region={option} zipCodeInput={zipCodeInput} />;
       }}
+      getOptionLabel={getOptionLabel}
```

Comment:
There was an error when selecting metro areas, this makes the error go away 🤷

Reply:
oh interesting. also- didnt see this comment [before writing mine below](https://github.com/covid-projections/covid-projections/pull/2416#discussion_r556987450). feel free to ignore

Reply:
making `getOptionLabel` return an empty string could nix the location-name-flicker, but not sure if that nixes the error you were seeing for metros